### ISSUE TITLE
TextEditor.editorElement is deprecated.

### ISSUE THREAD

**Issue by lexjacobs** [VoiceCode/atom-voicecode#18]
`TextEditor.prototype.editorElement` has always been private, but now
it is gone. Reading the `editorElement` property still returns a
reference to the editor element but this field will be removed in a
later version of Atom, so we recommend using the `element` property instead.
```
TextEditor.editorElement (<embedded>:2383:20)
<unknown> (/Users/aj/.atom/packages/voicecode/lib/voicecode.coffee:93:59)
CallbacksRegistry.apply (/Applications/Atom.app/Contents/Resources/electron.asar/common/api/callbacks-registry.js:48:42)
EventEmitter.<anonymous> (/Applications/Atom.app/Contents/Resources/electron.asar/renderer/api/remote.js:299:21)
emitThree (events.js:116:13)
EventEmitter.emit (events.js:194:7)
```

**Comment by lexjacobs** [comment]
I think this warning started appearing concurrent with the release of atom 1.19.1.

**Comment by lexjacobs** [comment]
This has resolved itself after updating to 1.19.1

**Comment by zach-nervana** [comment]
I am still receiving this warning using version 1.19.1
